cloud/aws/metrics/README.md

```diff
@@ -0,0 +1,3 @@
+# AWS Prometheus metrics
+
+This directory contains files for creating the Docker image for the Prometheus metrics scraper that runs alongside the CiviForm server on AWS ECS. The image is hosted on Docker hub and needs to be pushed there if any changes are made.
```

Comment:
Don't know if that's the best place to put it, but still. Can we put high-level description of monitoring components somewhere? Like who talks to whom, who exposes data. Or add link to your DD. 

Because while reviewing this PR I see variable `PROMETHEUS_WRITE_ENDPOINT` and wondering who is doing the writing, as I don't yet remember from top of my head what exactly prometheus does here.

Reply:
I'm planning to add a page to our docs after this PR and add a link to that docs page in this README. Would that be alright?

Reply:
Sure. sg